tests/custom_managers/models.py

```diff
@@ -194,3 +194,15 @@ class OneToOneRestrictedModel(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class AbstractPerson(models.Model):
+    class Meta:
```

Comment:
see https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style for model style guide (fields/managers before Meta)